### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by gabrielcoder247**
### `PAIRED-PORTFOLIO`
**In these milestone.**
I added the followings:

> HTML ( Created a nav, welcome-section and project-section)

> CSS (Created styling for the nav, welcome-section and project-section)

> LINTERS (created .stylintrc.json and .hintrc files)

> GITIGNORE (added node_modules to gitignore file)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+<body>
+    <div id="navbar">
+        <ul>
+            <li><a href="#welcome-section">Home</a></li>
+            <li><a href="#projects">News</a></li>
+            <li><a href="#contact">Contact</a></li>
+            <li><a href="#about">About</a></li>
+        </ul>
+    </div>
```
Comment (gabrielcoder247) [inline index.html:20]: Hey Gabriel :grinning: , you've got a great project here but you still need to make some little changes

Comment (gabrielcoder247) [reply index.html:20]: Use nav in the id section  `"navbar"` it make your work semantic and also professional

File: index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <section id="welcome-section">
+        <h1>Welcome to my Portfolio</h1>
+    </section>
+    <section id="projects">
+        <h2 class="project-title">Project 1 </h2>
+        <a href="https://cdn.pixabay.com/photo/2016/03/29/08/48/project-1287781_960_720.jpg"></a>
+    </section>
```
Comment (gabrielcoder247) [inline index.html:27]: Try to use more of class than `id's`, use id's when you have to select something unique and use class to group, also remember you can have more than one class in tag

**Review by gabrielcoder247 — COMMENTED** [review]
You're project is great try to make the required changes and you will be fine.

Once again Happy coding :smile:

**Comment by Davidosky007** [comment]
## Hi @gabrielcoder247 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
nice-job

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**